### PR TITLE
open dock at bottom by default

### PR DESCRIPTION
**Pull request by gokulprathin8**
1 line change to open the dock in bottom in sidepanel.

### CODE REVIEW

File: src/main/resources/static/js/gw.process.sidepanel.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ GW.process.sidepanel = {
             GW.process.sidepanel.showProcessLog(GW.process.sidepanel.current_workflow_history_id, 
                 GW.process.sidepanel.current_workflow_process_id, GW.process.sidepanel.current_process_name);
 
+            GW.process.sidepanel.bottomDock();
```
Comment (ZihengSun) [inline src/main/resources/static/js/gw.process.sidepanel.js:48]:
```suggestion
            GW.process.sidepanel.dockmode == "bottom";
            GW.process.sidepanel.renderDock();
```

**Review by ZihengSun — CHANGES_REQUESTED** [review]
You cannot directly call bottomDock because next time the bottom dock will be turned off

**Comment by gokulprathin8** [comment]
yes, the next time we press the bottom dock it will be disabled, is it not the expected behaviour? since we hide the dock on clicking the same button again for vertical align as well

dock alignment

**Comment by ZihengSun** [comment]
The behavior you mentioned is expected. But it will cause another unexpected behavior: if people close the side panel, and open it again. The bottom dock will disappear. It should always show bottom dock on the first open.

**Pull request merged**